test/_log.js

```diff
@@ -527,4 +528,15 @@ describe('Debug Logging Utility', () => {
     });
   });
 });
-
+describe('Testing manual update', () => {
+  it('Should return a string', () => {
+    v.update.updateManual('1.0.0', 'test');
+    expect('1.0.0');
```

Comment:
You aren't passing anything to the expect to compare it to. You should have the function above set to a const that you can compare this to.